docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<p>should we add the <code>process</code> function, perhaps t can show what happens to and input example for each step? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>